dashboard/conf/supervisor-app.conf

```diff
@@ -17,26 +17,6 @@
 # under the License.
 #
 
-[program:postgres]
-command=/usr/lib/postgresql/9.6/bin/postgres -D /etc/postgresql/9.6/main
-user = postgres
-autostart=true
-autorestart=true
-startsecs=5
-stopasgroup=true
-killasgroup=true
-stopsignal=TERM
-log_stdout=true
-log_stderr=true
-priority=500
-stdout_logfile=/var/log/supervisor/postgres.log
-stderr_logfile=/var/log/supervisor/postgres-error.log
-stdout_logfile_maxbytes=50MB
-stdout_logfile_backups=2
-stderr_logfile_maxbytes=50MB
-stderr_logfile_backups=2
```

Comment:
Seems all these config above is not needed any more?

Reply:
Correct, these were the supervisor config but we cannot start postgres with supervisor anymore. We need it to be up before, to initialize the db.